data/avro-schema/src/main/resources/avro/net/corda/data/certificates/CertificateType.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "enum",
+  "name": "CertificateType",
+  "namespace": "net.corda.data.certificates",
+  "doc": "A cluster level certificate type.",
```

Comment:
Is this going to be used generally or just for cluster-level certificates?

Reply:
Just for cluster-level certificates.